packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -741,6 +741,13 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     return binding.pipelineOwner.ensureSemantics();
   }
 
+  /// Flush the semantics.
```

Comment:
Why do you need this? A normal pump should be enough to trigger a semantics flush.